robyn/openapi.py

```diff
@@ -394,6 +394,16 @@ def get_schema_object(self, parameter: str, param_type: Any) -> dict:
                 properties["type"] = type_mapping[type_name]
                 return properties
 
+        # 检查是否为泛型类型（如List[Object]）
+        if hasattr(param_type, "__origin__"):
+            if param_type.__origin__ is list or param_type.__origin__ is List:
+                properties["type"] = "array"
+                # 处理列表中的元素类型
```

Comment:
Hey @yzutyc 👋 

Would it be possible for you to change your comments to English?